src/components/button/button.svelte

```diff
@@ -149,30 +149,35 @@
     font: var(--leo-font-components-button-small);
     --padding-y: 6px;
     --padding-x: 10px;
+    border-radius: 14px;
   }
   .leoButton.isSmall {
     --icon-size: 20px;
     font: var(--leo-font-components-button-small);
     --padding-y: 8px;
     --padding-x: 14px;
+    border-radius: 16px;
   }
   .leoButton.isMedium {
     --icon-size: 24px;
     font: var(--leo-font-components-button-default);
     --padding-y: 10px;
     --padding-x: 16px;
+    border-radius: 20px;
   }
   .leoButton.isLarge {
     --icon-size: 24px;
     font: var(--leo-font-components-button-large);
     --padding-y: 12px;
     --padding-x: 30px;
+    border-radius: 22px;
   }
   .leoButton.isJumbo {
     --icon-size: 24px;
     font: var(--leo-font-components-button-jumbo);
     --padding-y: 18px;
     --padding-x: 26px;
+    border-radius: 24px;
```

Comment:
Inspecting the button in devtools, the button is `60px` high, so should this be `30px`?